payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1794,6 +1796,48 @@ class Stripe internal constructor(
         }
     }
 
+    /**
+     * Retrieve a list of possible brands for the given card number.
+     * Returns an error if the cardNumber length is less than 6 characters.
+     *
+     * @param cardNumber the card number
+     * @param callback a [ApiResultCallback] to receive the result or error
+     */
+    @UiThread
+    fun retrievePossibleBrands(
+        cardNumber: String,
+        callback: ApiResultCallback<PossibleBrands>
+    ) {
+        CardNumber.Unvalidated(cardNumber).bin ?: run {
+            callback.onError(
+                InvalidRequestException(
```

Comment:
I didn't see a test for the callback variant of this exception.

Reply:
https://github.com/stripe/stripe-android/pull/6335/files#diff-b390689c9b65a1bfb2d7389f00d45fd819289a57e4e000bea116de59320629b5R1316-R1332